packages/terra-button/src/Button.jsx

```diff
@@ -243,6 +243,8 @@ class Button extends React.Component {
     } = this.props;
 
     const theme = this.context;
+    const isMac = () => navigator.userAgent.indexOf('Mac') !== -1 && navigator.userAgent.indexOf('Win') === -1;
```

Comment:
Why do we need to test for Win AND Mac? Shouldn't the first condition be sufficient?

Reply:
The first condition should be sufficient indeed. I saw that both terra-dropdown-button and terra-form-select have both conditions and I thought to keep the "pattern", but the second condition is probably redundant, so I'm removing it.